src/main/java/uk/gov/hmcts/reform/em/stitching/domain/EntityAuditEvent.java

```diff
@@ -42,6 +42,10 @@ public class EntityAuditEvent implements Serializable {
     @Column(name = "entity_value")
     private String entityValue;
```

Comment:
we can remove this from class?

Reply:
No. We can't remove this from class. Not untill data from this column is migrated to new column.